stream/src/lib.rs

```diff
@@ -1,3 +1,3 @@
 //! [Placeholder description]
```

Comment:
Waiting to approve until these are removed. I don't think we need to do #199 in this PR but could.